eng/scripts/get_package_properties.py

```diff
@@ -1,11 +1,74 @@
 import argparse
-import sys
-import glob
 import os
 import re
 
+from typing import Dict, List
+
 from ci_tools.parsing import ParsedSetup
 
+additional_pr_triggers: Dict[str, List[str]] = {
```

Comment:
This is the source of truth for dependent packages that the python team will maintain.